packages/manager/apps/dedicated/client/app/account/user/dashboard/user-dashboard.html

```diff
@@ -55,12 +55,24 @@
                     <div class="oui-tile__definition d-flex min-w-0">
                         <span class="oui-icon mr-3 oui-icon-user"></span>
                         <div class="word-break min-w-0">
-                            <div class="mb-4">
+                            <div
+                                data-ng-if=":: $ctrl.user.auth.method !== 'provider'"
```

Comment:
Two times used `provider`, better as constant:
```suggestion
                                data-ng-if=":: $ctrl.user.auth.method !== $ctrl.AUTH_METHOD_PROVIDER'"
```